src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState, useEffect } from "react";
+import "./home.css";
+import { BannerCard, Loader, ProductCard } from "../../components";
+import keyboardImg from "../../assets/banner-images/banner-keyboard.png";
+import xboxImg from "../../assets/banner-images/xbox-controller.png";
+import vrImg from "../../assets/banner-images/vr.png";
```

Comment:
In the asset folder, you can make an `index.js` file. And then in that, do `export {default as vrImg} from "..path"`. And then, here you can import like

`import {vrImg} from "../../assets"` 

In that way instead of importing images on three lines. You can import them on a single line.